packages/truffle-box/test/box.js

```diff
@@ -1,81 +1,104 @@
 
       // Now assert the README.md and the .gitignore file were removed.
       assert(fs.existsSync(path.join(destination, "README.md")) == false, "README.md didn't get removed!");
       assert(fs.existsSync(path.join(destination, ".gitignore")) == false, ".gitignore didn't get removed!");
     });
+  });
 
-    it("won't re-init if anything exists in the destination folder", function(done) {
-      this.timeout(5000);
-
-      const contracts_directory = path.join(destination, "contracts");
-
-      // Assert our precondition
-      assert(fs.existsSync(contracts_directory), "contracts directory should exist for this test to be meaningful");
-
-      fs.remove(contracts_directory).then(() => {
-        Box.unbox(TRUFFLE_BOX_DEFAULT, destination)
-          .then(function(boxConfig) {
-            assert(
-              fs.existsSync(contracts_directory) == false,
-              "Contracts directory got recreated when it shouldn't have"
-            );
-            done();
-          })
-          .catch(function(e) {
-            if (e.message.indexOf("Something already exists at the destination.") >= 0) {
-              done();
-            } else {
-              done(new Error("Unknown error received: " + e.stack));
-            }
-          });
-      });
-    });
+  describe("--force", () => {
 
-    it("will re-init if force is true in the options and files exist", function(done) {
-      this.timeout(5000);
+    it("unboxes truffle box when used", (done) => {
+      Box.unbox(TRUFFLE_BOX_DEFAULT, destination, { force: true })
+        .then(() => {
+          done();
+        });
+    }).timeout(5000);
+
+    it("runs without a prompt", (done) => {
+      Box.unbox(TRUFFLE_BOX_DEFAULT, destination, { force: true })
+        .then(() => {
+          assert.strictEqual(inquirer.prompt.called, false);
+          done();
+        });
+    }).timeout(5000);
 
+    it("overwrites redundant files if init/unbox force flag used", (done) => {
       const truffleConfigPath = path.join(destination, "truffle.js");
 
-      // Assert our precondition
-      assert(fs.existsSync(truffleConfigPath), "truffle.js should exist for this test to be meaningful");
+      // preconditions
+      fs.writeFileSync(truffleConfigPath, "this truffle.js file is different than the default box file", "utf8");
+      assert(fs.existsSync(truffleConfigPath), "mock truffle.js wasn't created!");
+      const mockConfig = fs.readFileSync(truffleConfigPath, "utf8");
+
+      Box.unbox(TRUFFLE_BOX_DEFAULT, destination, { force: true })
+        .then(() => {
+          assert(fs.existsSync(truffleConfigPath), "truffle.js wasn't recreated!");
+          const newConfig = fs.readFileSync(truffleConfigPath, "utf8");
+          assert(newConfig !== mockConfig, "truffle.js wasn't overwritten!");
+          done();
+        });
+    }).timeout(5000);
+
+  });
+
+  describe("init/unbox prompt", () => {
```

Comment:
I feel like you could probably just get rid of the last two tests in this group and just test that when 4 name conflicts exist, inquirer gets called 4 times.  But you probably should start the test by confirming that you do indeed have 4 name conflicts.